### ISSUE TITLE
Accessibility: keyboard trap in custom engrave dialog

### ISSUE THREAD

**Issue by mauritslamers** [frescobaldi/frescobaldi#1598]
Under Windows 10, a user depending on keyboard navigation alone is unable to use the `Tab` key to move the focus out of the "Additional Command Line options" edit field. 

While there are workarounds (`ctrl-return` to start engraving and `Esc` to close the dialog, it is confusing that `Tab` does not move the focus. It does not have any real advantage for entering command line options.

An additional issue is that the text field itself doesn't have the label above attached, so a screen reader simply read it out as "Edit".
